app/assets/css/style.css

```diff
@@ -15,67 +15,72 @@ a:hover {
 input[type="text"],
 input[type="password"] {
     background: white;
-    border: 1px solid #BFE6E2;
-    padding: 2px;
+    border: none;
+    padding: 15px 10px;
     font-family: 'Trebuchet MS', sans-serif;
-    font-size: 14px;
-    -moz-border-radius: 2px;
-    -webkit-border-radius: 2px;
-    color: #105751;
+    font-size: 16px;
+    color: black;
 }
 
 input[type="submit"] {
-    background: #105751;
-    border: 1px solid #073B36;
-    padding: 1px 3px;
+    background: #1eb4ff;
+    padding:  10px;
     font-family: 'Trebuchet MS', sans-serif;
-    font-size: 14px;
+    font-size: 16px;
     font-weight: bold;
-    -moz-border-radius: 2px;
-    -webkit-border-radius: 2px;
```

Comment:
It's always good to have them so we ensure it has the same look on Firefox and chrome ! (General comment for all the one you deleted below as well)

Reply:
We only need them, where we use `border-radius` though, feel free to make the changes in the PR (would be a good Git exercise ;))